src/components/templates/ExternalRoom/ExternalRoom.tsx

```diff
@@ -22,40 +36,74 @@ export const ExternalRoom: React.FC<ExternalRoomProps> = ({ venue }) => {
     openUrl(redirectUrl);
   }, [redirectUrl]);
 
-  return redirectUrl ? (
-    <div className="ExternalRoom">
-      <div className="ExternalRoom__message">
-        <SparkleLogo />
-        <div className="ExternalRoom__content">
-          <h4 className="ExternalRoom__header">
-            You will now be redirected to some incredible content! Please ensure
-            you{" "}
-            <a
-              rel="noreferrer"
-              href="https://support.google.com/chrome/answer/95472?hl=en&co=GENIE.Platform%3DDesktop"
-              target="_blank"
-            >
-              allow popups in your browser
-            </a>{" "}
-            and keep this tab open while you explore, so you can find your way
-            back easily. If you do not see a new tab open, please feel free to
-            click through to:{" "}
-            <a
-              rel="noreferrer"
-              href={venue.zoomUrl}
-              target="_blank"
-              className="ExternalRoom__link"
-            >
-              {venue.zoomUrl}
-            </a>
-            .
-          </h4>
+  const { worldUsers } = useWorldUsers();
+
+  return (
+    <div
+      className="ExternalRoom"
+      style={{ backgroundImage: `url(${EXTERNAL_ROOM_BACKGROUND})` }}
```

Comment:
`useCss()` ?

Reply:
example for quick reference https://github.com/sparkletown/sparkle/pull/2134/files#diff-117f5b50b7cbe0d164f27f6a0df1ab442ae728ce17a2f194b0ecb540e721c4bdR113-R117